src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -86,21 +89,26 @@ public override Task InitializeAsync()
 
             this.federationManager.Initialize();
 
+            if ((this.network.Consensus.Options as PoAConsensusOptions).VotingEnabled)
```

Comment:
Why not use direct cast if you do not expect `this.network.Consensus.Options` to be null.

Reply:
For no particular reason. 
Fixed